rennovate.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
added a config which is based on the hmpps rennovate config but configured to go at the first of the month.
(with a trial run for today)
 
This will probably need tweaking.

Reply:
This is how we structured the config for Renovate in my old team (don't know what we could learn from it)

https://github.com/ministryofjustice/opg-sirius/blob/main/.github/renovate.json